src/main/java/HereApi/JunctionTraversability.java

```diff
@@ -0,0 +1,56 @@
+package HereApi;
+
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * denotes, which junctions along a road are open
+ * can be:
+ * ALL_OPEN
+ * ALL_CLOSED
+ * INTERMEDIATE_CLOSED_EDGE_OPEN
+ * START_OPEN_OTHERS_CLOSED
+ * END_OPEN_OTHERS_CLOSED
+ */
+public enum JunctionTraversability
+
+{
+    ALL_OPEN, ALL_CLOSED, INTERMEDIATE_CLOSED_EDGE_OPEN, START_OPEN_OTHERS_CLOSED, END_OPEN_OTHERS_CLOSED;
```

Comment:
Steht in der Here-Api Dokumentation etwas zur Bedeutung der verschiedenen Status? Falls ja, bitte hier ergänzen

Reply:
gemacht